recipes-core/tdx-enc-handler/tdx-enc-handler/tdx-enc-caam.sh

```diff
@@ -17,9 +17,15 @@ TDX_ENC_STORAGE_MOUNTPOINT="@@TDX_ENC_STORAGE_MOUNTPOINT@@"
 # dm-crypt device to be created
 TDX_ENC_DM_DEVICE="encdata"
 
+# Flag to enable preservation of data on partition before encryption
+TDX_ENC_PRESERVE_DATA=@@TDX_ENC_PRESERVE_DATA@@
+
+# storage location of data backup file (if needed)
+TDX_ENC_PRESERVE_LOCATION="/tmp/encdata.tar.bz2"
```

Comment:
What about renaming this variable to `TDX_ENC_BACKUP_LOCATION` or `TDX_ENC_BACKUP_FILE`? I think it makes more clear to read the code.

Reply:
Will update to use the `TDX_ENC_BACKUP_FILE` in my next squash.